apps/dashboard/src/@/constants/env.ts

```diff
@@ -20,3 +20,6 @@ export const PROD_OR_DEV_URL = isProd ? "thirdweb.com" : "thirdweb-dev.com";
 export const DASHBOARD_STORAGE_URL =
   process.env.NEXT_PUBLIC_DASHBOARD_UPLOAD_SERVER ||
   "https://storage.thirdweb.com";
+
+export const API_SERVER_URL =
```

Comment:
as long as we're clear that this is only supposed to be used in backend functions and never directly from the client this is fine.